srcpkgs/m1n1/files/m1n1-kernel-hook.confd

```diff
@@ -0,0 +1,10 @@
+# can be either 1=kernel or 2=uboot (default)
+PAYLOAD=2
+# the location where kernel and initramfs are located
+BOOT=/boot
+U_BOOT=/usr/lib/asahi-boot/u-boot-nodtb.bin
+M1N1=/usr/lib/asahi-boot/m1n1.bin
+# the location where m1n1 payload is put
+TARGET=${BOOT}/m1n1/boot.bin
```

Comment:
shouldn't this be targeting /boot/efi/m1n1 ?

This will work if the ESP is mounted to /boot, but this is a nonstandard configuration

the asahi fedora reference mounts the ESP to /boot/efi

Reply:
you can change BOOT to /boot/efi in line 4, and it should work as well. I use /boot, which is also totally fine.